yajsapi/executor/strategy.ts

```diff
@@ -69,8 +69,10 @@ export class DummyMS extends MarketGeneral {
 
 export class LeastExpensiveLinearPayuMS {
   private _expected_time_secs: number;
-  constructor(expected_time_secs: number = 60) {
+  private _price_caps?: ComLinear;
+  constructor(expected_time_secs: number = 60, price_caps?: ComLinear) {
```

Comment:
`fixed_price_cap` in `ComLinear` is declared as `fixed_price!: number;` so it's supposed to be not `null` or `undefined`.
In the Python original, it's `float`, so not `None` either. Thus I'm not sure if using `ComLinear` to represent user constraints, which are optional, is a good idea: what if the user does not want to put any constraints on the fixed price?

Reply:
Perhaps it would be better to use two separate parameters: `max_fixed_price?: number` and `max_price_for?: Map<com.Counter, number>`?

Reply:
Fixed as you suggested.